sys/net/application_layer/gcoap/gcoap.c

```diff
@@ -712,6 +746,10 @@ void gcoap_register_listener(gcoap_listener_t *listener)
 
     listener->next = _coap_state.listeners;
     _coap_state.listeners = listener;
+
```

Comment:
Would be grouped more readably together with link_encoder which also sets a default.